code/api/src/modules/crateProduct/model.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+module.exports = function(sequelize, DataTypes) {
+  let CrateProduct = sequelize.define('crateProducts', {
+    productId: {
+      type: DataTypes.INTEGER
+    },
+    crateId: {
+      type: DataTypes.INTEGER
+    }
+  })
+
+  CrateProduct.associate = function(models) {
```

Comment:
I'm curious as to what this `associate` method does?

Reply:
The associate method allows us to declare the relationships between the different tables aka how they're associated with each other. This allows queries to pull information like a user's deliveries or a delivery's products through crateProducts (a joins table).